### ISSUE TITLE
Rename the new `App::add_systems` to `App::add_system`, deprecated compatibility breakage

### ISSUE THREAD

**Issue by nathanfranke** [bevyengine/bevy#8864]
## What problem does this solve or what need does it fill?

In my opinion, this makes system initialization look cleaner, makes adding single systems make grammatical sense, parity with singular methods `add_state`, `insert_resource`, etc.. I think of a tuple of systems as a _single_ system that runs multiple tasks in parallel (just as how the app systems run in parallel).

## What solution would you like?

Rename the new `App::add_systems(Schedule, Systems)` to `App::add_system(Schedule, System)`

I acknowledge this is a very visual change that may break compatibility for a common method. I personally think it is worth the rename.

Before:
```rs
.add_systems(Update, player_input)
.add_systems(Update, (drain_hunger, print_hunger))
```

After:
```rs
.add_system(Update, player_input)
.add_system(Update, (drain_hunger, print_hunger))
```

## What alternative(s) have you considered?

- "overload" the old deprecated method, keeping it and avoiding compatibility breakage. Might be possible with some [nightly crate](https://github.com/zenixls2/overloadf).
- Dealing with it

**Comment by alice-i-cecile** [comment]
This is something that @cart, @maniwani, @james7132 and I considered during the rename.

Personally, I'm moderately opposed. I don't think it's going to be obvious to users that multiple systems can be added via tuple-based overloading like this with the singular name.

I'm also very opposed to adding back a distinct `add_system` method: the change to "a single way to add systems" was absolutely for the better.

**Comment by cart** [comment]
I agree with @alice-i-cecile. A function that accepts one-to-many systems should be called `x_systems`.
